Server/App/api/deck.py

```diff
@@ -168,7 +170,7 @@ def put(self, name, public, tags, actions, parent_deck_version, parent_user_data
             # Increment both the deck version and the user data version.
             # TODO set last_update and last_update_device.
             connector.call_procedure('INCREMENT_DECK_VERSION', user_id, uuid)
-            connector.call_procedure('INCREMENT_USER_DATA_VERSION', user_id, uuid)
+            connector.call_procedure('INCREMENT_USER_DATA_VERSION', user_id, uuid, device)
```

Comment:
I'm wondering if we should also add the device to the create deck procedure as well